modules/installing-ocp-agent-ibm-z.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/installing-with-agent-based-installer.adoc
+// * installing/installing_with_agent_based_installer/prepare-pxe-infra-agent.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-ocp-agent-ibm-z_{context}"]
+= Manually adding {ibm-z-title} agents
```

Comment:
If this is only needed for IBM Z users, is there any way you could indicate this condition through either the title or the description? The method I've been using in the [main Agent installation page](https://docs.openshift.com/container-platform/4.14/installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html) is starting the heading with "Optional". It would look like this:

```suggestion
= Optional: Manually adding {ibm-z-title} agents
```

If you do this, the abstract below might need reframing to indicate this procedure as not necessary for regular users but necessary for IBM Z users. Another option would be something more conditional than optional like this:
```suggestion
= {ibm-z-title} only: Manually adding {ibm-z-title} agents
```

In any case I'll leave it up to you, I'd just prefer some indication that this isn't a required part of the workflow for non-IBM Z users.

Reply:
For IBM Z customers it's not optional. Therefore, I'll take your second suggestion.